pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -29,6 +29,20 @@ const (
 	minSupportedLibvirtVersion = "3.4.0"
 )
 
+func checkRunningInsideWSL2() error {
+	version, err := ioutil.ReadFile("/proc/version")
+	if err != nil {
+		return err
+	}
+
+	if strings.Contains(string(version), "Microsoft") {
+		logging.Debugf("Running inside WSL2 environment")
```

Comment:
Is this log needed ? If it fails just after, we should already see it in the log file.

Reply:
Consistency; following the same pattern as other preflight checks